src/Copy/BrandCopy.elm

```diff
@@ -34,6 +34,17 @@ brandCopy key =
                 , promptShort = "Contact"
                 }
 
+        CallToActionThree ->
+            CallToAction
+                { action = DesktopInteractive
+                , category = "stories"
+                , icon = "book"
+                , href = "#/stories"
+                , displayHref = ""
+                , promptLong = "Hope Stories"
+                , promptShort = "Hope Stories"
+                }
+
```

Comment:
The nav bar is populated in left to right order in `nav.elm`, so if we want them to stay the same stories will have to be CTA2 and the contact form CTA3.